desktop/apps/auction2/components/page/Footer.js

```diff
@@ -0,0 +1,80 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import block from 'bem-cn'
+import renderTemplate from 'desktop/components/react/utils/render_template'
+import { first } from 'underscore'
+
+export default function Footer (props) {
+  const { articles, auction, footerItems } = props
+  const footerItem = first(footerItems)
+  const showArticles = Boolean(articles.length)
+  const showFooterItems = footerItem && !auction.isAuctionPromo()
+  const b = block('auction-footer')
+
+  if (!showArticles) {
+    return null
+  }
+
+  return (
+    <footer
+      className={b({without: false})
+        .mix('auction-page-section')
+        .mix(articles.length ? 'has-articles' : 'has-no-articles')}
+    >
+      { showArticles &&
+        <div className={b('auction-articles')}>
+          { articles.models.map((article, key) => {
+            let articleFigureHTML
+
+            // Serverside
+            if (typeof window === 'undefined') {
+              articleFigureHTML = renderTemplate('desktop/components/article_figure/template.jade', { locals: { article, ...props } })
```

Comment:
🤔  I see why this serverside/clientside split is needed b/c of how browserify needs requires to be statically analyzable. I wonder though if we could instead change `renderTemplate`s API to be `renderTemplate(() => requrie('.../template.jade'), { locals: {} })` so that it defers on the serverside to avoid choking on .jade require and is still statically analyzable on the client-side. This is the woraround we use in testing too. Kinda ugly, but it avoids having to split the rendering at the component-level.